pkg/cmd/cli/cmd/startbuild.go

```diff
@@ -178,14 +178,8 @@ func RunStartBuild(f *clientcmd.Factory, out io.Writer, cmd *cobra.Command, args
 	}
 
 	wg.Wait()
-	// When waiting for the build to complete and the build fail, exit with 1 so the
-	// scripts can read this and treat the build as a failed.
-	if exitErr != nil {
-		fmt.Fprintf(cmd.Out(), "%v\n", exitErr)
-		os.Exit(1)
```

Comment:
Also worth mentioning here is that if `start-build foo --follow` fails for some reason, this will always return a nil error, though as discussed with @mfojtik printing --follow errors out is enough for now.